components/de_amp/browser/de_amp_throttle.cc

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/de_amp/browser/de_amp_throttle.h"
+
+#include <utility>
+
+#include "base/feature_list.h"
```

Comment:
many of these headers don't appear to be used. Please also make sure to remove any unused deps after removing them